app/Main.hs

```diff
@@ -51,7 +51,7 @@ contracts = ContractRegistry $ Map.fromList
     { ciContract = holdingsContract
     , ciIsDocumented = True
     , ciStorageParser = Just holdingsStorageParser
-    , ciStorageNotes = H.storageNotes
+    , ciStorageNotes = Just H.storageNotes
```

Comment:
Why can't we use auto-generated notes? They are supposed to be good enough. How do they compare to `H.storageNotes`?

Reply:
Right, I forgot that we now have autogenerated storage notes

Reply:
The only difference is datatype constructor name prefix, e.g. `%sfTokenMeta` instead of `%tokenMeta`